contracts/core/UniversalChannelHandler.sol

```diff
@@ -45,6 +45,15 @@ contract UniversalChannelHandler is IbcReceiverBaseUpgradeable, UUPSUpgradeable,
         __IbcReceiverBase_init(_dispatcher);
     }
 
+    /**
+     * @dev Close a universal channel.
+     * Cannot send or receive packets after the channel is closed.
+     * @param channelId The channel id of the channel to be closed.
+     */
+    function closeChannel(bytes32 channelId) external onlyOwner {
+        dispatcher.channelCloseInit(channelId);
+    }
```

Comment:
this should be part of the previous PR (ch close)

Reply:
ah yes - good catch. I added this change to[ this commit ](https://github.com/open-ibc/vibc-core-smart-contracts/pull/100/commits/53cb9230c2bd9fdb82f0cf8cebaa3a2967372690)on the v2 branch, so it should be included there instead now